pkg/server/server.go

```diff
@@ -67,7 +68,7 @@ func New(configurators ...Configurator) *Server {
 	srv.router.Post("/receive", srv.handleReceive)
 	srv.router.Post("/request-login", srv.handleLogin)
 	srv.router.Post("/authenticate", srv.handleAuthenticate)
-	srv.router.With(srv.requireAuthMiddleware).Route("/manage", func(r chi.Router) {
+	srv.router.With(middleware.NoCache, srv.requireAuthMiddleware).Route("/manage", func(r chi.Router) {
```

Comment:
Thank you :) I think I'd even extend that to any API endpoint.

Reply:
Sure :relaxed: